jenkins_job_wrecker/modules/handlers.py

```diff
@@ -173,6 +173,9 @@ def scmcheckoutretrycount(top, parent):
 def customworkspace(top, parent):
     parent.append(['workspace', top.text])
 
+def childcustomworkspace(top, parent):
```

Comment:
please create separate PRs for childcustomworkspace and buildnamesetter, with tests for each